src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py

```diff
@@ -78,17 +63,23 @@ def openvino_compile_cached_model(cached_model_path, *example_inputs):
         om.inputs[idx].get_node().set_partial_shape(PartialShape(list(input_data.shape)))
     om.validate_nodes_and_infer_types()
 
-    core.set_property({'CACHE_DIR': cache_root_path() + '/blob'})
+    core.set_property({'CACHE_DIR': _get_cache_dir(options) + '/blob'})
```

Comment:
What if user provided `CACHE_DIR`  in `config`? You will rewrite it, can be confusing to users

Reply:
Made changes so that CACHE_DIR is set appropriately